schema/config_blobset_blob.json

```diff
@@ -11,13 +11,20 @@
       "type": "string"
     },
     "base64": {
-      "type": "string"
+      "type": "string",
+      "pattern": "^[0-9A-Za-z=+/]{4,}$"
     },
     "url": {
-      "type": "string"
+      "type": "string",
+      "format": "uri"
     },
     "sha256": {
-      "type": "string"
+      "type": "string",
+      "pattern": "^[0-9A-Za-z]{64}$"
     }
-  }
+  },
+  "oneOf": [
+    { "required": ["content_type", "base64"] },
```

Comment:
I would have expected oneOf { base64, url } (transport mechanism), and then sha256 is always required, and then content_type is optional?  I suppose sha256 isn't required with base64... But doesn't content_type potentially apply to all transport mechanisms? Oh wait, is it because content_type is included in the HTTP response?

Reply:
Yes, content_type comes from http so we don't want to allow overriding it--feels dangerous to me